packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -148,10 +173,7 @@ class RenderParagraph extends RenderBox {
   }
 
   void _layoutText({ double minWidth: 0.0, double maxWidth: double.INFINITY }) {
-    _textPainter.layout(
-        minWidth: minWidth,
-        maxWidth: _softWrap || _overflow == TextOverflow.ellipsis ? maxWidth : double.INFINITY
-    );
+    _textPainter.layout(minWidth: minWidth, maxWidth: softWrap || overflow == TextOverflow.ellipsis ? maxWidth : double.INFINITY);
```

Comment:
Maybe pull maxWidth into a local for readability?

Reply:
pulled out the boolean expression into a local called "widthMatters".